src/pages/NotFoundPage/NotFoundPage.module.scss

```
@@ -2,4 +2,199 @@
 
 .page {
   @extend %center;
+  height: 100vh;
+  flex-direction: column;
+  
+  /* colors */
+  $blue: #293b49;
+  $pink: #fdabaf;
+  $pink-light: #ffe0e6;
+  $green: #04cba0;
+  $green-dark: #01ac88;
+  $white: white;
+
+  a {
+    text-decoration: none;
+  }
+
+  .center {
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  .error {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: center;
+  }
+
+  .number {
+    font-weight: 900;
```

```suggestion
    font-weight: 800;
```

У нас есть шрифты 400, 500, 600, 800. Я так понимаю жирность 900 просто не сработает, или вместо неё сработает 800